### ISSUE TITLE
No status even though game is recognized 

### ISSUE THREAD

**Issue by 0wg** [andrewburgess/steamdeck-discord-status#18]
Decky v3.0.1
Discord Status v1.4.0
Discord Stable 333428

Plugin says connected, and it's recognizing my game, but my status does not show up on discord.

**Comment by Johnny365365** [comment]
I have the same issue. Can't pinpoint the timeframe, but it seems to have happened as of this morning.  

Discord Status 1.3.1

Something must have be broke on the server end. 

Thanks for all the work you do for the plugin.

**Comment by gonzo-oin** [comment]
Exact same issue here.

**Comment by cameronkeys** [comment]
Would love for this to be fixed

**Comment by Johnny365365** [comment]
It works again for me as of last week. No issues and the games are detected on Discord. Forgot to update my post. If yours still isn't working, maybe make sure Discord is up to date, and remove/install Discord Status in Decky Plugins again? Other than that, I don't know other troubleshooting steps, as it is working fine for me.

**Comment by cameronkeys** [comment]
Thanks for letting me know, I hadn’t updated yet

**Comment by cameronkeys** [comment]
Yeah no fix on my end. Still shows in steam deck UI the game but on discord nothing at all is being played

**Comment by midbizkit** [comment]
2 months later and I'm still having this issue 😓

**Comment by andrewburgess** [comment]
Not sure if I'll be able to resolve, but if you wouldn't mind grabbing the logs and sharing them here, maybe I can figure it out.

> [!WARNING]
> The logs will have your Discord user name and ID, so you'll probably want to remove or redact those before pasting them here if that concerns you

The log file names should be dates when they were started, so try to grab the most recent one after updating the plugin to the latest version.

Unless otherwise configured, they should be in `~/homebrew/logs/DiscordStatus`

Again, no guarantees because this plugin is a hack to try to make it work so there's only so much I'll be able to do

**Comment by midbizkit** [comment]
sorry I'm late but it's working now! thank you for your reply

**Comment by ssxpreme** [comment]
Feb 19 2026 - worked great for 24hr, now doesnt work. Game recognized on status plug-in, no longer showing status despite working fine prior.
